Daniel Migault's editorial feedback (#81)
lamps-wg/csr-attestation: draft-ietf-lamps-csr-attestation.md

rooted in a device manufacture trust anchor and the leaf certificate being
on the device in question; the latter is the Attestation Key that signs the Evidence statement.

> This document specifies a CSR Attribute (or Extension for CRMF CSRs) for carrying evidence. An evidence statement in an existing format can be placed into an EvidenceStatement along with an OID to identify its type and optionally a hint to the Relying Party about how to verify it. A set of EvidenceStatements may be grouped together along with the set of CertificateAlternatives needed to validate them to form a EvidenceBundle. One or more EvidenceBundles may be placed into the id-aa-evidenceStatement CSR Attribute (or CFRM Extension).

Reviewer: ```suggestion
This document specifies a CSR Attribute (or Extension for Certificate Request Message Format (CRMF) CSRs) for carrying Evidence. Evidence can be placed into an EvidenceStatement along with an OID to identify its type and optionally a hint to the Relying Party about where to verify it. A set of EvidenceStatements may be grouped together along with the set of CertificateAlternatives needed to validate them to form a EvidenceBundle. One or more EvidenceBundles may be placed into the id-aa-evidenceStatement CSR Attribute (or CRFM Extension).
```

Author: I included this change manually.

 .------------.               .----|----------.
 |            +-------------->|----'          | Compare Attestation
 |  Attester  |   Evidence    | Relying       | Result against

>  |   (HSM)    |   in CSR      | Party (RA/CA) | policy

Reviewer 2: ```suggestion
 |  (with HSM)    |   in CSR      | Party (RA/CA) | policy
```

Reviewer 2: The HSM is not the Attstester, just a part of one of the Attester's Attesting Environments
